src/main/java/sg/ncl/MainController.java

```diff
@@ -128,6 +127,11 @@ public String pricing() {
         return "pricing";
     }
 
+    @RequestMapping("/resources1")
+    public String resources1() {
+        return "resources1";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this method and declare a constant for this value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3400)